services/src/handlers/workflows.rs

```diff
@@ -677,6 +675,7 @@ mod tests {
     use actix_web::{http::header, http::Method, test};
     use actix_web_httpauth::headers::authorization::Bearer;
     use geoengine_datatypes::collections::MultiPointCollection;
+    use geoengine_datatypes::dataset::NamedData;
```

Comment:
not sure where to put this comment but: on GET /workflow the system namespace should be omitted as well, or not?

Reply:
Is this part of an external provider or is this the internal one?

Reply:
it's the mock provider. I guess we can't omit the system namespace because then we wouldn't know whether the uuid is the namespace or the provider right?